main.go

```diff
@@ -5,8 +5,19 @@ import (
 )
 
 func main() {
-	// this is no purpose comment and should trigger a review comment
-	// fmt.Prinln("hello world")
+	// this comment doesnt add any value and should trigger a review comment instead..
```

Comment:
Remove the comment that doesn't add any value and should trigger a review comment instead.